Gemfile

```diff
@@ -18,6 +18,7 @@ group :development, :test do
   gem 'faker', '~>3.1.1'
   gem 'rspec-rails'
   gem 'rubocop-rails', require: false
+  gem 'shoulda-matchers', '~> 5.0'
```

Comment:
Since we are only using this gem for testing, add it to the `group :test` so it's not installed in other environments.